moveit_ros/planning/planning_request_adapter_plugins/res/default_plan_request_adapter_params.yaml

```diff
@@ -0,0 +1,44 @@
+default_plan_request_adapter_parameters:
+  path_tolerance: {
+    type: double,
+    description: "AddTimeOptimalParameterization: Tolerance per joint in which the time parameterization is allowed to deviate from the original path.",
+    default_value: 0.1,
+  }
+  resample_dt: {
+    type: double,
+    description: "AddTimeOptimalParameterization: Time steps between two waypoints of the parameterized trajectory. The trajectory is re-sampled from the original path.",
+    default_value: 0.1,
+  }
+  min_angle_change: {
```

Comment:
Canonical term for this might be `angle_epsilon` or something?

Reply:
I think `min_angle_change` is more descriptive, so I'd be in favor of keeping it